aws/resource_aws_codebuild_project_test.go

```diff
@@ -551,6 +551,41 @@ func TestAccAWSCodeBuildProject_Source_GitCloneDepth(t *testing.T) {
 	})
 }
 
+func TestAccAWSCodeBuildProject_Source_GitSubmodulesConfig(t *testing.T) {
```

Comment:
Excellent new test case. 💯 It would be wonderful to see a similar one added to verify secondary sources as well. 👍

Reply:
New test case added: `TestAccAWSCodeBuildProject_SecondarySources_GitSubmodulesConfig`